### PR TITLE
AO3-5791 Restrict width of user-inserted images

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5791

## Purpose

Makes it so images in works, admin posts, etc can only be 100% of the container's width to avoid overflow/horizontal scrolling on screens smaller than the images. 

## Testing Instructions

Refer to Jira.

## References

~Don't merge until the post is ready -- hence "Deferred or Superseded."~ Post is ready! Just needs to go out a few days before deploy.

### CODE REVIEW

File: public/stylesheets/site/2.0/21-userstuff.css [hunk 1]
```diff
@@ -139,6 +139,11 @@
   border: 1px solid;
 }
 
+.userstuff img {
+  max-width: 100%;
+  height: auto;
```
Comment (redsummernight) [inline public/stylesheets/site/2.0/21-userstuff.css]: Is this to preserve images' aspect ratios?

Comment (sarken) [reply public/stylesheets/site/2.0/21-userstuff.css]: Yup! It'll override any `height` attribute the user set on the `img` tag to make sure nothing gets stretched out.

**Comment by sarken** [comment]
Noticed there was a version of this just for news posts, so I removed that after merging with master.

**Comment by sarken** [comment]
Closing so I can get under 5 pull requests and open a new one for an issue that's blocking two of my existing PRs.

**Pull request merged**